### ISSUE TITLE
Contrast for h3 in related links (mobile)

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#705]
WCAG2AA
Discovered with: http://squizlabs.github.io/HTML_CodeSniffer/

```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.19:1. Recommendation: change background to #555451.
```

Code Snippet:

```
<h3>Related Links</h3>
```

I only found this issue while browsing on mobile
